OMG.Classroom/DataAccessLayer/Seeder/Seed.cs

```diff
@@ -0,0 +1,308 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccessLayer.Seeder
+{
+    public class Seed
```

Comment:
Make this class static since we won't manage some state here - it just has static methods